CHANGELOG.md

```diff
@@ -19,6 +19,12 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Introduce experimental searchable snapshot API ([#4680](https://github.com/opensearch-project/OpenSearch/pull/4680))
 - Introduce Remote translog feature flag([#4158](https://github.com/opensearch-project/OpenSearch/pull/4158))
 - Add groupId value propagation tests for ZIP publication task ([#4848](https://github.com/opensearch-project/OpenSearch/pull/4848))
+- Recommissioning of zone. REST layer support. ([#4624](https://github.com/opensearch-project/OpenSearch/pull/4604))
+- Make searchable snapshot indexes read-only but allow deletion ([#4764](https://github.com/opensearch-project/OpenSearch/pull/4764))
```

Comment:
This line should be the only change in the this file

Reply:
Agreed and I had the same thought when following the backporting instructions on my original PR. It came after I cherry picked my original PR commit
<img width="960" alt="image" src="https://user-images.githubusercontent.com/10157886/198374425-1b3b5540-8fb7-4c3f-8b1a-8bd3a24cf887.png">

Reply:
Yeah you should just pick the line of the change you're backporting. There's a lot of contention on the CHANGELOG file so this is tedious, but hopefully we can improve this process.

Reply:
Updated